src/pkg/packager/common.go

```diff
@@ -496,14 +496,18 @@ func (p *Packager) validatePackageArchitecture() (err error) {
 	return nil
 }
 
-// validateLastNonBreakingVersion compares the Zarf CLI version against a package's LastNonBreakingVersion.
-// It will return an error if there is an error parsing either of the two versions,
-// and will throw a warning if the CLI version is less than the LastNonBreakingVersion.
+// validateLastNonBreakingVersion validates the Zarf CLI version against a package's LastNonBreakingVersion.
 func (p *Packager) validateLastNonBreakingVersion() (err error) {
 	cliVersion := config.CLIVersion
 	lastNonBreakingVersion := p.cfg.Pkg.Build.LastNonBreakingVersion
 
-	if lastNonBreakingVersion == "" || cliVersion == "UnknownVersion" {
+	if lastNonBreakingVersion == "" {
+		return nil
+	}
+
+	if cliVersion == "unset" || cliVersion == "UnknownVersion" {
```

Comment:
I don't think we explicitly need this as a separate check since the language it returns would suffice for an invalid semantic version as well - generally fewer if blocks means easier to understand logic.

Reply:
Removed this check